spec/mailers/previews/user_mailer_preview.rb

```diff
@@ -190,6 +190,17 @@ def in_person_ready_to_verify_reminder
       email_address: email_address_record,
     ).in_person_ready_to_verify_reminder(
       enrollment: in_person_enrollment,
+      is_enhanced_ipp: false,
+    )
+  end
+
+  def in_person_ready_to_verify_reminder_enhanced_ipp_enabled
```

Comment:
If you navigate to `localhost:3000/rails/mailer`, you should now be able to select the "in_person_ready_to_verify_reminder_enhanced_ipp_enabled" preview from the list.